src/register-org/components/organisation-services-access/organisation-services-access.component.html

```diff
@@ -7,12 +7,12 @@
       <form [formGroup]="servicesFormGroup">
         <div class="govuk-form-group">
           <fieldset class="govuk-fieldset" aria-describedby="changed-name-hint">
-            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-              <h1 class="govuk-fieldset__heading">
+            <legend>
```

Comment:
@johnbenjamin-hmcts - revert changes to the `<legend>` class and change the last one to the `--xl` variant

Reply:
@Daniel-Lam - I have changed the code so that the header and spacing match with other pages.

Reply:
@Daniel-Lam - I have updated the code now.